ClipboardAPI/clipboard-change-event-explainer.md

```diff
@@ -0,0 +1,197 @@
+      - [2.2.2 Copy plain text should show only single paste option in Excel online](#222-copy-plain-text-should-show-only-single-paste-option-in-excel-online)
+      - [2.2.3 Multiple paste options in Google sheets](#223-multiple-paste-options-in-google-sheets)
+  - [3. Motivation - Alternative to inefficient polling of clipboard](#3-motivation---alternative-to-inefficient-polling-of-clipboard)
+  - [4. Example javascript code for detecting clipboard changes:](#4-example-javascript-code-for-detecting-clipboard-changes)
+  - [5. Event spec details and open questions](#5-event-spec-details-and-open-questions)
+    - [5.1 User permission requirement](#51-user-permission-requirement)
+      - [5.1.1 Approach 1 (Preferred) - clipboard-read permission required to listen to clipboardchange event](#511-approach-1-preferred---clipboard-read-permission-required-to-listen-to-clipboardchange-event)
+        - [Pros](#pros)
+        - [Cons](#cons)
+      - [5.1.2 Approach 2 - No permission required](#512-approach-2---no-permission-required)
+        - [Pros](#pros-1)
+        - [Cons](#cons-1)
+      - [5.1.3 Conclusion](#513-conclusion)
+    - [5.2 Page focus requirement](#52-page-focus-requirement)
+      - [5.2.1 Approach 1 (Preferred) - Page required to be in focus to receive event](#521-approach-1-preferred---page-required-to-be-in-focus-to-receive-event)
+        - [Pros](#pros-2)
+        - [Cons](#cons-2)
+      - [5.2.2 Approach 2 - No focus requirement](#522-approach-2---no-focus-requirement)
+        - [Pros:](#pros-3)
+        - [Cons:](#cons-3)
+      - [5.2.3 Approach 3 - Transient user activation](#523-approach-3---transient-user-activation)
+        - [Pros:](#pros-4)
+        - [Cons:](#cons-4)
+      - [5.2.4 Conclusion](#524-conclusion)
+    - [5.3 Event details](#53-event-details)
+  - [6 Appendix](#6-appendix)
+    - [6.1 APIs provided by all OS to listen to clipboardchange event:](#61-apis-provided-by-all-os-to-listen-to-clipboardchange-event)
+  - [7 References \& acknowledgements](#7-references--acknowledgements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## 1. Introduction
+
+The clipboardchange event fires whenever the system clipboard contents are changed. This allows web-apps like remote desktop clients to be notified and respond to changes to the system clipboard. It provides an efficient alternative to polling the clipboard for changes.
+
+## 2. User scenarios
+
+### 2.1 Scenario: Sync clipboard with a remote desktop
+When a user copies text or an image on their local machine, the web-based remote desktop application can detect that clipboard contents have changed by listening for the `clipboardchange` event. Upon detecting the change, the application can re-read the clipboard and send the updated clipboard content to the remote desktop environment.
```

Comment:
Might be good to add in this scenario that the page is focused (as concluded in 5.2), otherwise it may sound like you're changing focus requirements for this API.

Reply:
Makes sense, I've added this point